pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -519,19 +509,28 @@ func (c *Controller) syncCheckStates(ctx context.Context, wl *kueue.Workload, ch
 				"accepted", isAccepted(pr),
 				"bookingExpired", isBookingExpired(pr),
 				"capacityRevoked", isCapacityRevoked(pr))
+			retryStrategy := ptr.Deref(prc.Spec.RetryStrategy, kueue.DefaultRetryStrategy)
+			backoffBaseSeconds := *retryStrategy.BackoffBaseSeconds
+			backoffMaxSeconds := *retryStrategy.BackoffMaxSeconds
+			backoffLimitCount := *retryStrategy.BackoffLimitCount
```

Comment:
```suggestion
```

As I commented above, once we use the static marker defauler, we do not need this variables. We can just directly use those fields.